x/basket/spec/protobuf.md

```diff
@@ -0,0 +1,207 @@
+ <!-- This file is auto-generated. Please do not modify it yourself. -->
```

Comment:
We're not creating an `x/basket` module

Reply:
bug in protocgen.sh -> https://github.com/regen-network/regen-ledger/blob/release/v2.2.x/scripts/protocgen.sh#L43
Will crerate a task for that

Reply:
okay, it's probably not something we really need to worry about